web/src/lib/components/asset-viewer/asset-viewer.svelte

```diff
@@ -232,9 +221,11 @@
 		isShowAlbumPicker = false;
 		const album = event.detail.album;
 
-		api.albumApi
-			.addAssetsToAlbum(album.id, { assetIds: [asset.id] })
-			.then((response) => showAddNotification(response.data));
+		addAssetsToAlbum(album.id, [asset.id]).then((dto) => {
+			if (dto.successfullyAdded === 1 && dto.album) {
+				appearsInAlbums = [...appearsInAlbums, dto.album];
```

Comment:
Can we add the `notificationController.show` method here to indicate that the asset has been added sucessfully to the album?

Reply:
That was the idea of the `addAssetsToAlbum` function. It already sends the notification. Or did you mean something else?

Reply:
Hmm, I don't see the notification showing up on the asset-viewer page. Hmm, probably due to the z-index.

Reply:
I either cannot replicate the issue or am not sure where you don't see the notification. Can you please tell me what steps are you taking when you don't see the notification?

Reply:
1. Click on the photo to view it in detailed view
2. Click on the vertical three dots at the upper top right and select "Add to album"
3. Select an album to add the photo in
4. You don't see the notification saying that the photo is added to the album

Reply:
We can ignore this since this probably unrelated to this PR